sdk/apis/third_party/conditions/apis/conditions/v1alpha1/constants.go

```diff
@@ -132,9 +132,9 @@ const (
 )
 
 const (
-	// MachineHealthCheckSuccededCondition is set on machines that have passed a healthcheck by the MachineHealthCheck controller.
+	// MachineHealthCheckSucceededCondition is set on machines that have passed a healthcheck by the MachineHealthCheck controller.
 	// In the event that the health check fails it will be set to False.
-	MachineHealthCheckSuccededCondition ConditionType = "HealthCheckSucceeded"
+	MachineHealthCheckSucceededCondition ConditionType = "HealthCheckSucceeded"
```

Comment:
Let's drop those changes because they live in a `third_party` directory, which means the code has been imported from some upstream project. In this case, it's CAPI (#273). We shouldn't change third party code if we can avoid it. I'm not sure if this condition is actually used anywhere in our code, probably not, but changes on our end might make it more difficult to pull upstream code again.